jobserver/management/commands/cancel_jobs.py

```diff
@@ -0,0 +1,37 @@
+"""Management command to cancel Jobs via the RAP API."""
+
+import structlog
+from django.core.management.base import BaseCommand
+
+from jobserver import rap_api
+
+
+class Command(BaseCommand):
+    """Management command to cancel Jobs via the RAP API."""
+
+    help = "Cancel RAP jobs via the RAP API."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "job_request_id",
```

Comment:
I know we're building in inconsistency, but
```suggestion
            "rap_id",
```

Reply:
Yes, let's mirror the remote API params. And the `help_text` can hint about the situation.